components/board/CommentInput.tsx

```diff
@@ -0,0 +1,76 @@
+import { useAuth } from "@/contexts/AuthProvider";
+import styles from "@/styles/CommentInput.module.scss";
+import axios from "@/lib/axios";
+import { useRouter } from "next/router";
+import { ChangeEvent, FormEvent, useState } from "react";
+import { getCookie } from "cookies-next";
+
+interface CommentInputProps {
+  articleId: string;
+}
+
+export default function CommentInput({ articleId }: CommentInputProps) {
+  const [content, setContent] = useState("");
+  const [isAble, setIsAble] = useState(true);
+  const { user } = useAuth();
+  const router = useRouter();
+
+  const handleChange = (e: ChangeEvent<HTMLInputElement>) => {
+    const nextValue = e.target.value;
+    setContent(nextValue);
```

Comment:
@ssseeo0 

이렇게 분리하기보단 
```suggestion
  const { value } = e.target;
  setContent(value)
```
로 하는게 조금 더 좋을 것 같아요.

이미 있는 객체 필드를 destructuring 하는게 새로운 변수를 만들어 스택메모리 사용하는것보다는 더 효율성이 좋지 않을까요?